karma_config/vueLocal.js

```diff
@@ -0,0 +1,35 @@
+const vue = require('../node_modules/vue');
```

Comment:
This is pretty much copied from our core component, but without the async loading of the Intl polyfill. Not sure if it is worth abstracting in the core component constructor to keep it DRY, but obeying the rule of three for now :)